system-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/identityhub/IdentityHubIntegrationTest.java

```diff
@@ -42,28 +47,57 @@ public class IdentityHubIntegrationTest {
             .readTimeout(1, TimeUnit.MINUTES)
             .build();
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final ConsoleMonitor CONSOLE_MONITOR = new ConsoleMonitor();
 
     private IdentityHubClientImpl client;
 
     @BeforeEach
     void setUp() {
-        client = new IdentityHubClientImpl(OK_HTTP_CLIENT, OBJECT_MAPPER);
+        client = new IdentityHubClientImpl(OK_HTTP_CLIENT, OBJECT_MAPPER, CONSOLE_MONITOR);
     }
 
     @ParameterizedTest
     @MethodSource("provideHubUrls")
-    void retrieveVerifiableCredentials_empty(String hubUrl) throws IOException {
+    void retrieveVerifiableCredentials(String hubUrl, String region) {
+        await().atMost(20, SECONDS).untilAsserted(() -> singleVcInIdentityHub(hubUrl));
+
+        singleVcInIdentityHub(hubUrl)
+                .satisfies(jwt -> {
+                    var claims = jwt.getJWTClaimsSet();
+                    assertThat(claims.getIssuer()).as("Issuer is a Web DID").startsWith("did:web:");
+                    assertThat(claims.getSubject()).as("Subject is a Web DID").startsWith("did:web:");
```

Comment:
nit: does it make sense to get issuer as env var and match the full issuer here?

Just a question rather a change comment,

Reply:
In the next PR, we will introduce the `IdentityHubCredentialsVerifier` which will bring this verification. I think we can adapt the test then.